downloader/src/main/kotlin/vcs/Subversion.kt

```diff
@@ -226,15 +226,15 @@ class Subversion : VersionControlSystem() {
             }
 
             true
-        } catch (e: SVNException) {
-            e.showStackTrace()
+        }.onFailure {
+            it.showStackTrace()
```

Comment:
commit-msg: Would it make sense to mention that the DownloadExceptions are not changed?

Reply:
I'm not exactly sure what you're after, but doesn't "or any occurred exception" imply that?

Reply:
Right.